src/providers/facebook-login-provider.ts

```diff
@@ -1,18 +1,19 @@
 import { BaseLoginProvider } from "../entities/base-login-provider";
 import { SocialUser } from "../entities/user";
+import { LoginOpt } from '../auth.service';
 
 declare let FB: any;
 
 export class FacebookLoginProvider extends BaseLoginProvider {
 
   public static readonly PROVIDER_ID: string = "FACEBOOK";
 
-  constructor(private clientId: string, private scope = 'email,public_profile') { super(); }
+  constructor(private clientId: string, private opt: LoginOpt = { scope: 'email,public_profile'}, private locale: string = 'en_US') { super(); }
```

Comment:
What else other than scope could go here? Otherwise having only the scope would be sufficient.

Reply:
Fb.login method may contain auth_type, scope, return_scopes, enable_profile_selector etc..
You may refer it at [Fb.login opts](https://developers.facebook.com/docs/reference/javascript/FB.login/v2.11).

I've included Options interfaces inside [auth.service.ts](https://github.com/abacritt/angularx-social-login/pull/42/files/0bb654f7034d7d154d59e629005203b397144f11#diff-fc44d3e9dbd69a283d10a34bedfb0443) files that you might need to refer too.